explorer/client/src/components/displaybox/DisplayBox.tsx

```diff
@@ -75,29 +93,57 @@ function DisplayBox({ display }: { display: string }) {
         );
     } else {
         return (
-            <div className={styles['display-container']} id="displayContainer">
-                {!hasDisplayLoaded && (
-                    <div className={styles.imagebox} id="pleaseWaitImage">
-                        Image Loading...
+            <>
+                {hasClickedImage && (
+                    <div
```

Comment:
One thing I'd think about for the future is making this modal accessible. Modals like this need to trap focus, have specific a11y labelling requirements, must be keyboard controllable, etc. Ideally we can just use a library to manage all of the modal complexity for us.

Reply:
Feel free to make specific recommendations regarding modal libraries and making the modals more accessible. Be aware as to the design requirements specific to the Figma Design.